go.sum

```diff
@@ -0,0 +1,136 @@
+github.com/bep/clocks v0.5.0 h1:hhvKVGLPQWRVsBP/UB7ErrHYIO42gINVbvqxvYTPVps=
```

Comment:
Remove any unused dependencies to keep the module clean and avoid potential security vulnerabilities. Unused libraries can clutter the codebase and lead to maintenance issues.